src/tests/ReactNoDangerousHtmlRuleTests.ts

```diff
@@ -34,9 +33,8 @@ function someFunction() {
                 },
                 {
                     failure:
-                        'Invalid call to dangerouslySetInnerHTML in method "<unknown>"\n    of source file ' +
-                        Utils.absolutePath('file.tsx') +
-                        '"\n    Do *NOT* add a suppression for this warning. ' +
+                        'Invalid call to dangerouslySetInnerHTML in method "<unknown>".' +
+                        '\n    Do *NOT* add a suppression for this warning. ' +
```

Comment:
Added `.` to the end of previous line. And removed non-matched `"` after file name (probably copy-paste os string with `\n`)